packages/vue-dot/src/patterns/FormBuilder/mixins/formBuilderCore.ts

```diff
@@ -68,6 +68,7 @@ export class FormBuilderCore extends MixinsDeclaration {
 	/**
 	 * When the form is updated, emit a
 	 * change event with updated data
+	 * Also emits 'refresh' event if the field is dynamic
```

Comment:
```suggestion
	 * Also emit 'refresh' event if the field is dynamic
```
On utilise `emit` plus haut